packages/es-components/src/components/patterns/datepicker/DatePicker.js

```diff
@@ -12,16 +11,16 @@ import Textbox from '../../controls/textbox/Textbox';
 class DateTextbox extends React.Component {
```

Comment:
Can this be a function now that we don't need to utilize componentDidMount?

Reply:
I wonder if we even need this DateTextbox now that we've got the refs sorted out in Textbox. Could Textbox just include the focus() method now?

Reply:
We can't remove the `DateTextbox` class in the short term. `react-date-picker` needs to be able to place a ref on whatever we pass to the `customInput`. Refs will not work on functional components. In order to remove `DateTextbox` completely we will need to use the `forwardRef` api. I put a sample of that [here](https://github.com/WTW-IM/es-components/commit/44db02388461d7d12e8288308ae968ec1719652e). Using `forwardRef` is on hold until we change the minimum react version to `16.3.0`.